C/plugins/storage/sqlite/plugin.cpp

```diff
@@ -40,8 +41,37 @@ const char *default_config = QUOTE({
 			"default" : "5",
 			"displayName" : "Pool Size",
 			"order" : "1"
-			}
-		});
+		},
+		"nReadingsPerDb" : {
+			"description" : "Number of readings tables per  database",
+			"type" : "integer",
+			"default" : "15",
+			"displayName" : "N Readings per database",
+			"order" : "2"
+		},
+		"nDbPreallocate" : {
+			"description" : "Number of databases to allocate in advance, NOTE: SQLite has a maximum number of attachable databases by default at 10",
+			"type" : "integer",
+			"default" : "3",
+			"displayName" : "N databases to allocate in advance",
+			"order" : "3"
+		},
+		"nDbLeftFreeBeforeAllocate" : {
+			"description" : "Number of databases left free before a new allocation is executed",
```

Comment:
"Allocate new databases when the number of free databases drops below this value"

Reply:
done